pairing/bls12381/circl/suite_test.go

```diff
@@ -8,15 +8,12 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"go.dedis.ch/kyber/v3"
-	this "go.dedis.ch/kyber/v3/pairing/circl_bls12381"
 	"go.dedis.ch/kyber/v3/sign/bls"
 	"go.dedis.ch/kyber/v3/sign/tbls"
 	"go.dedis.ch/kyber/v3/sign/test"
 	"go.dedis.ch/kyber/v3/util/random"
 )
 
-// Code extracted from kyber/utils/test
-// TODO: expose API in forked drand/kyber
```

Comment:
actually if this code is already in kyber/utils/test, couldn't we re-use the functions from there??